.github/workflows/maven.yml

```diff
@@ -19,12 +16,15 @@ jobs:
         cache: maven
     - name: Build with Maven
       run: mvn -B package -DskipTests --file pom.xml
-
-    - name: Publish Docker Image
+    - name: Publish Docker Images
       run: |
           # login to GitHub Container Registry
           echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${GITHUB_ACTOR} --password-stdin
-          # build image
-          docker build . --tag ghcr.io/eclipse-opensmartclide/smartclide/security:$(date +'%Y-%m-%d')
-          # push image
-          docker push ghcr.io/eclipse-opensmartclide/smartclide/security:$(date +'%Y-%m-%d')
\ No newline at end of file
+          # build images
+          docker build -t ghcr.io/eclipse-opensmartclide/opensmartclide/ssas:$(date +'%Y-%m-%d') -t ghcr.io/eclipse-opensmartclide/opensmartclide/ssas:latest .
+          docker build -t ghcr.io/eclipse-opensmartclide/opensmartclide/vam:$(date +'%Y-%m-%d') -t ghcr.io/eclipse-opensmartclide/opensmartclide/vam:latest -f VulnerabilityAssessmentModels/Dockerfile .
+          docker build -t ghcr.io/eclipse-opensmartclide/opensmartclide/sonarqube-security-opensmartclide:$(date +'%Y-%m-%d') -t ghcr.io/eclipse-opensmartclide/opensmartclide/sonarqube-security-opensmartclide:latest -f sonarqube/Dockerfile .
+          # push images
+          docker push ghcr.io/eclipse-opensmartclide/opensmartclide/ssas --all-tags
+          docker push ghcr.io/eclipse-opensmartclide/opensmartclide/vam --all-tags
+          docker push ghcr.io/eclipse-opensmartclide/opensmartclide/sonarqube-security-opensmartclide --all-tags
```

Comment:
All required images are pushed to the GitHub registry. When pulling the images, the tag :latest should be used.

Reply:
> It appears that the GH action is stuck again, and there are no images in the registry yet.
> 
> Contact eclipse (again) to fix the action?

Yeah, I'll open a ticket.

Reply:
@philipreimer I opened [this issue](https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/2693).

Reply:
The publish_image check in main branch settings was turned of by Eclipse temporarily